index.html

```diff
@@ -49,5 +47,183 @@ <h1 class="intro__heading">
         </div>
 
     </section>
+     
+    <section class="container__card">
+      <figure class="card">
+            <figcaption class="card__img--cont">
+              
+             
+  <img src="./images/SnapshootPortfolio.png" alt="image of a project" class="img__card">
+                <h2 class="desc__project">Tonic</h2>
+              <ul class="project__info--cont">
+                <li class="project__info--title">canopy</li>
+                
+                <li class="project__info--dot">
+                    <i class="fas fa-circle color__circle"></i>
+                </li>
+                <li class="project__info--author">Back End Dev</li>
+                 <li class="project__info--dot">
+                  <i class="fas fa-circle color__circle"></i>
+                 </li>
+                
+                <li class="project__info--date">2015</li>
+              </ul>
+            
+            <ul class="list__lng--btn">
```

Comment:
- [ ] Unlike in other project contents you appear to be missing the project description contained in a `p` tag here. Please add it.